airflow/contrib/hooks/gcp_speech_to_text_hook.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from google.cloud.speech_v1 import SpeechClient
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+
+class GCPSpeechToTextHook(GoogleCloudBaseHook):
```

Comment:
Docstring is missing.

Reply:
All hooks (except `GoogleCloudBaseHook`) for the GCP platform have very limited descriptions. What do you think should be included in this description? I am interested in the development of integration with the GCP platform and your opinion is valuable to me. What parts should the description consist of? 

https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_vision_hook.py#L87-L90
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_transfer_hook.py#L33-L37
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_spanner_hook.py#L27-L34
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_pubsub_hook.py#L40-L45
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_mlengine_hook.py#L48-L51
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_kms_hook.py#L38-L42
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_function_hook.py#L36-L42
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_function_hook.py#L36-L42
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_dataproc_hook.py#L206-L213
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_dataflow_hook.py#L176-L178
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_container_hook.py#L37-L42
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_compute_hook.py#L41-L46
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_bigtable_hook.py#L28-L34

Reply:
Hi @mik-laj , AWS-related hooks can be considered as reference, e.g. https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/aws_dynamodb_hook.py#L24-L34 .

The main missing piece to me here is the parameters and param types. Let's take https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_transfer_hook.py#L33-L37 as example: there are three parameters, `api_version`, `gcp_conn_id`, and `delegate_to`. For `api_version` & `delegate_to`: what's their usage/definition/purpose/valid values, as well as their types. These should be clear in the docstring IMO.

Reply:
Params description added

Reply:
Thanks for explaining. I will prepare a fix for all GCP hooks.